### PR TITLE
feat(workflow-manager): added workflow-manager to deis-dev manifests

### PR DESCRIPTION
**Pull request by jackfrancis**
Voici: `workflow-manager`

### CODE REVIEW

File: deis-dev/manifests/deis-workflow-manager-rc.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: WORKFLOW_MANAGER_PORT
+          value: "8080"
+        - name: WORKFLOW_MANAGER_VERSIONSAPIURL
+          value: "https://versions.deis.com"
+        - name: WORKFLOW_MANAGER_STAGINGAPIURL
+          value: "https://versions-staging.deis.com"
+        - name: WORKFLOW_MANAGER_DOCTORAPIURL
```
Comment (arschles) [inline deis-dev/manifests/deis-workflow-manager-rc.yaml:33]: this and the previous 2 env vars look like they're missing a few underscores. is that purposeful?

Comment (arschles) [inline deis-dev/manifests/deis-workflow-manager-rc.yaml:33]: same with a few of the ones below

File: deis-dev/manifests/deis-workflow-manager-rc.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    app: deis-workflow-manager
+  template:
+    metadata:
+      labels:
+        app: deis-workflow-manager
+    spec:
+      serviceAccount: deis-workflow-manager
+      containers:
+      - name: deis-workflow-manager
+        image: quay.io/deisci/workflow-manager:v2-beta
```
Comment (arschles) [inline deis-dev/manifests/deis-workflow-manager-rc.yaml]: I don't see a `v2-beta` tag at https://quay.io/repository/deisci/workflow-manager?tag=latest&tab=tags

File: deis-dev/manifests/deis-workflow-manager-rc.yaml (hunk 1 above)
Comment (jackfrancis) [inline deis-dev/manifests/deis-workflow-manager-rc.yaml:33]: Yeah, it was to make incorporation with https://github.com/kelseyhightower/envconfig a bit easier.

E.g., here's its usage in `workflow-manager`:

`envconfig.Process("workflow_manager", &Spec)`

`Spec` is a pointer to this:

```
// Specification config struct
type Specification struct {
    Port           string `default:"8080"`
    Polling        int    `default:"43200"` // 43200 seconds = 12 hours
    VersionsAPIURL string
    DoctorAPIURL   string
    APIVersion     string
    CheckVersions  bool   `default:"true"`
    DeisNamespace  string `default:"deis"`
}
```

When I used underscores in the property names, my golang IDE was complaining that that isn't the go way.

That's the purpose!

File: deis-dev/manifests/deis-workflow-manager-rc.yaml (hunk 2 above)
Comment (jackfrancis) [inline deis-dev/manifests/deis-workflow-manager-rc.yaml]: This is true. I am working with @sgoings to get that sorted w/ Jenkins et al.

File: deis-dev/manifests/deis-workflow-manager-rc.yaml (hunk 1 above)
Comment (arschles) [inline deis-dev/manifests/deis-workflow-manager-rc.yaml:33]: @jackfrancis ok, sounds good. don't think any changes are necessary then

nb: there is a `envconfig:"MY_ENV_VAR"` struct tag that you can use, which makes envconfig look for `MY_ENV_VAR` and assign its value to that field.

File: deis-dev/manifests/deis-workflow-manager-rc.yaml (hunk 2 above)
Comment (arschles) [inline deis-dev/manifests/deis-workflow-manager-rc.yaml]: ok, sounds good.

**Comment by arschles** [comment]
LGTM pending the resolution described at https://github.com/deis/charts/pull/149#discussion_r55615669

File: deis-dev/manifests/deis-workflow-manager-rc.yaml (hunk 1 above)
Comment (jackfrancis) [inline deis-dev/manifests/deis-workflow-manager-rc.yaml:33]: Awesome:

https://github.com/deis/workflow-manager/issues/10

Comment (arschles) [inline deis-dev/manifests/deis-workflow-manager-rc.yaml:33]: :+1:

File: deis-dev/manifests/deis-workflow-manager-rc.yaml (hunk 2 above)
Comment (sgoings) [inline deis-dev/manifests/deis-workflow-manager-rc.yaml]: We'll actually want to specify the latest `git-<sha>` in here. So as of right now that'd be `git-941cb10` (from https://quay.io/repository/deisci/workflow-manager?tab=tags). @vdice will help you craft the Jenkins job that will keep that sha up to date without any manual effort.

**Comment by jackfrancis** [comment]
@sgoings Updated to latest git-sha-tagged image.

**Pull request merged**